docs/source/new-tutorials/tutorial08.ipynb

```diff
@@ -208,7 +208,7 @@
     "collapsed": false
    },
    "source": [
-    "For the above method we used a quantile regression model, that uses the [pinball loss function](https://www.lokad.com/pinball-loss-function-definition) to assess the goodness-of-fit. While the defined coverage is guaranteed for the training date, it is not guaranteed for unseen data. More complex methods like Conformal Prediction can be used to guarantee coverage for unseen data. For a deeper dive into the topic, you can check out our tutorial on Uncertainty Quantification."
+    "For the above method we used a quantile regression model, that uses the [pinball loss function](https://www.lokad.com/pinball-loss-function-definition) to assess the goodness-of-fit. While the defined coverage is guaranteed for the training date, it is not guaranteed for unseen data. More complex methods like Conformal Prediction can be used to guarantee marginal coverage for unseen data. For a deeper dive into the topic, you can check out our tutorial on Uncertainty Quantification."
```

Comment:
Recommend adding this link on **marginal coverage**.

https://arxiv.org/pdf/1903.04684.pdf

Reply:
```suggestion
    "For the above method we used a quantile regression model, that uses the [pinball loss function](https://www.lokad.com/pinball-loss-function-definition) to assess the goodness-of-fit. While the defined coverage is guaranteed for the training date, it is not guaranteed for unseen data. More complex methods like Conformal Prediction can be used to guarantee [marginal coverage](https://arxiv.org/pdf/1903.04684.pdf) for unseen data. For a deeper dive into the topic, you can check out our tutorial on Uncertainty Quantification."
```